src/pages/workspaces/List.js

```diff
@@ -282,15 +273,61 @@ export const WorkspaceList = () => {
                   side: 'left'
                 }, [
                   Utils.switchCase(lastRunStatus,
-                    ['success', () => icon('success-standard', { size: 20, style: { color: colors.success() } })],
-                    ['failure', () => icon('error-standard', { size: 20, style: { color: colors.danger(0.85) } })],
-                    ['running', () => icon('sync', { size: 20, style: { color: colors.success() } })]
+                    ['success', () => icon('success-standard', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Success'
+                    })],
+                    ['failure', () => icon('error-standard', {
+                      size: 20, style: { color: colors.danger(0.85) },
+                      'aria-label': 'Workflow Status Failure'
+                    })],
+                    ['running', () => icon('sync', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Running'
+                    })]
                   )
                 ])
               ])
             ])
           },
           size: { basis: 30, grow: 0, shrink: 0 }
+        }, {
+          headerRenderer: () => div({ className: 'sr-only' }),
+          cellRenderer: ({ rowIndex }) => {
+            const { workspace: { cloudPlatform } } = sortedWorkspaces[rowIndex]
+            return div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
+              div({ style: styles.tableCellContent }, [
+                Utils.switchCase(cloudPlatform,
+                  ['Gcp', () => h(CloudGcpLogo, { title: 'Google Cloud Platform', role: 'img' })],
```

Comment:
I think it would look better if the cloud environment and success/failure icons were the same size.

![image](https://user-images.githubusercontent.com/484484/183921147-04e07a81-2d57-49e6-82ed-a27e2ca41fe0.png)

Reply:
I tried ` style: { height: 20 } ` and `size: 20` (and also numbers greater than 20), but they all seem to show the same image size. I wasn't sure how to make the cloud environment icons bigger--is there another way to change the image size?
`style: { height: 20 }`
<img width="100" alt="Screen Shot 2022-08-10 at 10 26 03 AM" src="https://user-images.githubusercontent.com/91842048/183926630-6507db15-5e6a-4b42-abe9-3bba5a88d5db.png">
`style: { height: 60 }`
<img width="110" alt="Screen Shot 2022-08-10 at 10 24 16 AM" src="https://user-images.githubusercontent.com/91842048/183926189-8758e6d6-54fb-40d9-bb0e-29afda45005d.png">